Freshworks-Samples/Omni-App-Samples/Custom-Installation-Page_Omni/app/app.js

```diff
@@ -0,0 +1,23 @@
+var client;
+
+var errorLogger = function (error) {
+  console.error("The following Error occured -", error);
```

Comment:
```suggestion
  console.error("The following Error occured.");
  console.error(error);
```

Since `error` is an object. Appending it with a string is not a good idea.
Instead you can append the error message alone with the string if the error object properties are always constant.